### PR TITLE
Add docker

### PR DESCRIPTION
**Pull request by ChataingT**
The script to compile version and versionTransition has been put inside a docker.
This docker is based on python.

The use should be where it's placed. It needs to be with :  
 * compute_versiondata_from_py3dfiles.py  for the execution of the script
 * data folder to find the tileset.json and the json's schema

### CODE REVIEW

File: Utils/ExtractVersionFromTilsesetJSON/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM python:3
+LABEL vendor="UD-CPOV project"
```
Comment (jailln) [inline Utils/ExtractVersionFromTilsesetJSON/Dockerfile:2]: Please change to "UD-SV project"

File: Utils/ExtractVersionFromTilsesetJSON/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+COPY requirement.txt ./
+
+RUN pip install --no-cache-dir -r requirement.txt
+
+COPY ./compute_versiondata_from_py3dfiles.py ./compute_versiondata_from_py3dfiles.py
+
+CMD [ "python", "compute_versiondata_from_py3dfiles.py"]
+# docker run --mount src=`pwd`,target=/Output,type=bind -it test_python:latest
```
Comment (jailln) [inline Utils/ExtractVersionFromTilsesetJSON/Dockerfile:11]: Is this line needed ?

File: Utils/ExtractVersionFromTilsesetJSON/readme.md [hunk 3]
```diff
@@ -10,16 +10,22 @@ This script should :
...
 You can install them with the command `pip install -r requirement.txt` 
 
-## Execution :  
+## Execution with python:  
 To execute the script, follow the requirement and then execute  "compute_versiondata_from_py3dfiles.py"
 
+## Execution with docker:  
+in this folder : 
+```
+$docker build -t compute_versionData .
```
Comment (jailln) [inline Utils/ExtractVersionFromTilsesetJSON/readme.md:24]: When executing this I get `invalid argument "compute_versionData" for "-t, --tag" flag: invalid reference format: repository name must be lowercase` and the docker is not built. Changing the tag to `compute_version_data` does the trick. Can you update the doc please ?

Comment (ChataingT) [reply Utils/ExtractVersionFromTilsesetJSON/readme.md]: Changed

File: Utils/ExtractVersionFromTilsesetJSON/Dockerfile (hunk 2 above)
Comment (ChataingT) [reply Utils/ExtractVersionFromTilsesetJSON/Dockerfile]: Taken out

File: Utils/ExtractVersionFromTilsesetJSON/Dockerfile (hunk 1 above)
Comment (ChataingT) [reply Utils/ExtractVersionFromTilsesetJSON/Dockerfile]: Changed

**Pull request merged**